receiver/sqlserverreceiver/scraper_test.go

```diff
@@ -20,6 +20,16 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatatest/pmetrictest"
 )
 
+func enableAllScraperMetrics(cfg *Config) {
+	cfg.MetricsBuilderConfig.Metrics.SqlserverDatabaseIoReadLatency.Enabled = true
+
```

Comment:
What is this empty line for?

Reply:
I thought it might be helpful to split metrics here by query they're coming from. So each "block" of metrics here is a different query. Just personal preference though, we can remove empty lines if that's the preference 👍